app/models/runtime/process_model.rb

```diff
@@ -99,6 +99,7 @@ def buildpack_type
         [:spaces, :guid, :organization_id]
       ]
 
+    # Nick: seems important
```

Comment:
Were these comments kept in by accident or did y'all have questions here?

@selzoc && maryam